src/lib/init.ts

```diff
@@ -790,6 +813,77 @@ function getGatewayPackageJSON(isJS: boolean) {
   };
 }
 
+function getGatewayServerless(isJS: boolean) {
+  return `${
+    isJS
+      ? ""
+      : 'import type { TypeBoxTypeProvider } from "@fastify/type-provider-typebox";\n'
+  }import fastify from "fastify";
+import dotenv from "dotenv";
+
+// Runtime dependencies required for this function. Vercel does import only direct listed dependencies.
+(() => [
+  import("@fluencelabs/js-client"),
+  // This import will fail in runtime
+  () => import("@fluencelabs/marine-worker"),
+  import("@fluencelabs/marine-js"),
+  // This imports will impact loading speed
```

Comment:
```suggestion
  // These imports will impact loading speed
```
Also I don't know whcih ones? all of them listed after that?